src/components/comments/CommentEditor.tsx

```diff
@@ -68,7 +68,7 @@ export const CommentEditor = (props: Props) => {
   }
 
   const onTxFailed: TxFailedCallback = () => {
-    ipfsCid && ipfs.removeContent(ipfsCid).catch(err => new Error(err))
+    ipfsCid && ipfs.unpinContentFromIpfs(ipfsCid).catch(err => new Error(err))
```

Comment:
Use `remoceContent` in all places because now we are using offchain server for it, and it is method for working with offchain

Reply:
@olehmell Is it okay to use it even if it's marked as deprecated?
![Screenshot 2023-03-31 at 19 27 49](https://user-images.githubusercontent.com/14356826/229120087-7a1e477a-77eb-4a22-9a37-4d4f65f3302b.png)

Reply:
Yes, because we dont recommend use it for anyone, but our app works with it now

Reply:
@olehmell Okay, understood. Changed to `removeContent` in [a475f49](https://github.com/dappforce/polkaverse/pull/80/commits/a475f4932bac6a52838d80f3f04a8d5315c295d6)